proto/feeburner/genesis.proto

```diff
@@ -11,6 +11,7 @@ option go_package = "github.com/neutron-org/neutron/x/feeburner/types";
 // GenesisState defines the feeburner module's genesis state.
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
-  TotalBurnedNeutronsAmount total_burned_neutrons_amount = 2 [ (gogoproto.nullable) = false ];
+  TotalBurnedNeutronsAmount total_burned_neutrons_amount = 2
```

Comment:
@faddat not quite sure here — did the linter do the changes in proto files, or did you do them manually?

Reply:
I don't think that I did that manually so I think that was probably make proto all